example/plugins/c-api/client_context_dump/client_context_dump.cc

```diff
@@ -116,7 +116,7 @@ dump_context(const char *ca_path, const char *ck_path)
 
         // Serial number
         int64_t sn = 0;
-#if OPENSSL_VERSION_NUMBER >= 0x010100000
+#if !defined(OPENSSL_IS_BORINGSSL) && OPENSSL_VERSION_NUMBER >= 0x010100000
```

Comment:
Just a nit, but your conditional in cert_reporting_tool.cc has braces around OPENSSL_VERSION_NUMBER comparison and this one doesn't.

Reply:
Added parens.